src/App.js

```diff
@@ -3,20 +3,29 @@ import './App.scss';
 
 import todos from './api/todos';
 import users from './api/users';
+import { TodoList } from './components/TodoList/TodoList';
+
+const fullList = todos.map(todo => ({
+  user: users.find(user => todo.userId === user.id),
+  ...todo,
+}));
 
 function App() {
   return (
     <div className="App">
       <h1>Static list of todos</h1>
-      <p>
-        <span>Todos: </span>
-        {todos.length}
-      </p>
+      <div className="App__subtitle">
+        <p>
+          <span>Todos: </span>
+          {todos.length}
+        </p>
 
-      <p>
-        <span>Users: </span>
-        {users.length}
-      </p>
+        <p>
+          <span>Users: </span>
+          {users.length}
+        </p>
+      </div>
+      <TodoList list={fullList} />
```

Comment:
list is too generic name

```suggestion
      <TodoList todos={fullList} />
```